Python/pythonrun.c

```diff
@@ -1199,6 +1199,38 @@ print_exception_notes(struct exception_print_context *ctx, PyObject *notes)
     return -1;
 }
 
+static int
+get_exception_notes(struct exception_print_context *ctx, PyObject *value, PyObject **notes) {
+    PyObject *type = NULL;
+    PyObject *errvalue = NULL;
+    PyObject *tback = NULL;
```

Comment:
These declarations can be inside the ``if (_PyObject_LookupAttr ...``. Also, I don't think they need to be initialised. ``PyErr_Fetch`` always sets them, right?

Reply:
Ah yes, I changed that now. `PyErr_Fetch` sets them, in other places where `PyErr_Fetch` is called these variables are not initialized.